src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -40,6 +40,7 @@
 #include "rest/ConnectionInfo.h"
```

Comment:
CNR entry missing (probably not a new entry, but expand the one that you already include in the first PR)